### ISSUE TITLE
nvda doesn't speak context menu in downloads and favorites in microsoft edge

### ISSUE THREAD

**Issue by Nikita34196** [nvaccess/nvda#15060]
I noticed that the last time for about a month nvda does not read the context menu in downloads and favorites in microsoft edge. installed the latest version of nvda alfa for today the latest version of microsoft edge.
windows11 system

**Comment by Qchristensen** [comment]
I'm going to close this as a duplicate of #14916 - which means we are already aware of the issue, and it would be worth going to that issue and activating the "Subscribe" button to be kept up to date with progress.

It is also important when creating an issue to follow the template, which would have got a little more detail on the specifics (exact steps to reproduce, which build of Windows 11, etc)
